docs/admin/installation/hardware.rst

```diff
@@ -451,9 +451,9 @@ NUC before it can be used. We recommend:
 
 .. _nucc12_recommendation:
 
-Intel 12th-gen NUC
-~~~~~~~~~~~~~~~~~~
-We have tested and can recommend the `Intel NUC12WSKi5 <https://ark.intel.com/content/www/us/en/ark/products/121625/intel-nuc-12-pro-mini-pc-nuc12wski5.html>`__.
+12th-gen NUC
```

Comment:
Renaming the section headers will also change the anchors, so we may want to add the old anchors for b/c with old links.

Reply:
Done.